### PR TITLE
MakeStyles refactor 1-3

### PR DESCRIPTION
**Pull request by andreas-unleash**
Signed-off-by: andreas-unleash <andreas@getunleash.ai>

MakeStyles refactor 1-3

## About the changes

Closes #

### CODE REVIEW

File: frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx [hunk 1]
```diff
@@ -18,6 +17,36 @@ export interface IAutocompleteBoxOption {
...
+        alignItems: 'center',
+        justifyContent: 'center',
+        paddingLeft: 6,
+        borderTopLeftRadius: '40px',
+        borderBottomLeftRadius: '40px',
+        color: theme.palette.text.tertiaryContrast,
+    })
+);
+
+const StyledAutocomplete = styled(Autocomplete)({
```
Comment (kwasniew) [inline frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx:46]: I'd do sx={styledAutocomplete} where styledAutocomple = {flex: 1}

Comment (kwasniew) [reply frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx:46]: Then typecasting won't be needed

File: frontend/src/component/common/Announcer/AnnouncerElement/AnnouncerElement.tsx [hunk 2]
```diff
@@ -1,25 +1,33 @@
 import React, { ReactElement } from 'react';
-import { useStyles } from 'component/common/Announcer/AnnouncerElement/AnnouncerElement.styles';
 import { ANNOUNCER_ELEMENT_TEST_ID } from 'utils/testIds';
+import { styled } from '@mui/material';
 
 interface IAnnouncerElementProps {
     announcement?: string;
 }
 
+const StyledContainer = styled('div')(() => ({
```
Comment (kwasniew) [inline frontend/src/component/common/Announcer/AnnouncerElement/AnnouncerElement.tsx:9]: unnecessary function

**Review by kwasniew — APPROVED** [review]

File: frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx [hunk 3]
```diff
@@ -18,6 +17,36 @@ export interface IAutocompleteBoxOption {
...
+const StyledContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: theme.spacing(2),
+    '& .MuiInputLabel-root[data-shrink="false"]': {
+        top: 3,
+    },
+}));
+
+const StyledIcon = styled('div')<{ disabled: boolean }>(
```
Comment (sjaanus) [inline frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx:29]: Proably should use **shouldForwardProp** as other similar components. Search from project for examples.

**Pull request merged**